test/get_transaction.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var Client = require('../');
+var assert = require('assert');
+var account_info = require('./fixtures/account_info')();
+var common = require('./common/common');
+var fixture = require('./fixtures/get_transaction');
+
+describe('Ripple REST Client Get Transaction', function() {
+  var client;
+  var payments;
+
+  before(function (done) {
+    client = new Client({
+      account: account_info.source_account
+    });
+
+    client.getPayments(null, function(error, payment_array) {
+      payments = payment_array;
+      done();
+    });
+  });
+
+  // Given the hash for the second last payment, the response should contain the
+  // hashes of last payment and third last payment.
+  it('should get notifications', function(done) {
+    this.timeout(5000);
+    client.getTransaction(payments[0].hash, function(error, response) {
```

Comment:
Might want to double check this but if more payments get added to that particular wallet it's possible that payments[0] might be different in the future.